op-batcher/batcher/service.go

```diff
@@ -260,6 +260,10 @@ func (bs *BatcherService) initChannelConfig(cfg *CLIConfig) error {
 		return fmt.Errorf("unknown data availability type: %v", cfg.DataAvailabilityType)
 	}
 
+	if bs.UseAltDA && cc.UseBlobs {
```

Comment:
Nit: the auto DA selection mode is covered by `cc.UseBlobs`, but it would be great if we could make this clearer somehow.

Reply:
I modified the error msg a bit in [fb93ea8](https://github.com/ethereum-optimism/optimism/pull/15126/commits/fb93ea839794f4f9bf115e40b060fa713b48734e).

Reply:
That helps! Thanks